pkg/sdn/plugin/subnets.go

```diff
@@ -266,13 +268,14 @@ func (plugin *OsdnNode) updateVXLANMulticastRules(subnets hostSubnetMap) {
 	otx := plugin.ovs.NewTransaction()
 
 	// Build the list of all nodes for multicast forwarding
-	tun_dsts := ""
+	tun_dsts := make([]string, 0, len(subnets))
 	for _, subnet := range subnets {
 		if subnet.HostIP != plugin.localIP {
-			tun_dsts += fmt.Sprintf(",set_field:%s->tun_dst,output:1", subnet.HostIP)
+			tun_dsts = append(tun_dsts, fmt.Sprintf(",set_field:%s->tun_dst,output:1", subnet.HostIP))
 		}
 	}
-	otx.AddFlow("table=111, priority=100, actions=move:NXM_NX_REG0[]->NXM_NX_TUN_ID[0..31]%s,goto_table:120", tun_dsts)
+	sort.Strings(sort.StringSlice(tun_dsts))
```

Comment:
ok, I say send a PR to fix in master and we'll allow the double casting since it apparently doesn't hurt much.